Error in server file while having MultipartRequest() in the service

```
# account/gen/http/user/server
account/gen/http/user/server/server.go:93:77: not enough arguments in call to NewUserUploadPicDecoder
	have (UserUploadPicDecoderFunc)
	want ("github.com/obo/am/vendor/goa.design/goa/http".Muxer, UserUploadPicDecoderFunc)
```

--- a/cors/generate.go
+++ b/cors/generate.go
@@ -149,7 +149,7 @@ func ServerCORS(f *codegen.File) {
 	for _, s := range f.Section("server-init") {
 		s.Source = strings.Replace(s.Source,
 			"e.{{ .Method.VarName }}, mux, {{ if .MultipartRequestDecoder }}{{ .MultipartRequestDecoder.InitName }}(mux, {{ .MultipartRequestDecoder.VarName }}){{ else }}dec{{ end }}, enc, eh",
-			`{{ if ne .Method.VarName "CORS" }}e.{{ .Method.VarName }}, mux, {{ if .MultipartRequestDecoder }}{{ .MultipartRequestDecoder.InitName }}({{ .MultipartRequestDecoder.VarName }}){{ else }}dec{{ end }}, enc, eh{{ end }}`,
+			`{{ if ne .Method.VarName "CORS" }}e.{{ .Method.VarName }}, mux, {{ if .MultipartRequestDecoder }}{{ .MultipartRequestDecoder.InitName }}(mux, {{ .MultipartRequestDecoder.VarName }}){{ else }}dec{{ end }}, enc, eh{{ end }}`,
 			-1)
 	}
 	for _, s := range f.Section("server-handler") {

